src/modules/GroupChannel/components/Message/MessageView.tsx

```diff
@@ -239,6 +239,16 @@ const MessageView = (props: MessageViewProps) => {
 
   const renderedCustomSeparator = useMemo(() => renderCustomSeparator?.({ message }) ?? null, [message, renderCustomSeparator]);
 
+  const prevShouldRenderSuggestedReplies = useRef(shouldRenderSuggestedReplies);
```

Comment:
Nice!! I could not think of this: useRef can recognize height changes of a component.